test/codegen/__snapshots__/codegen.js.snap

```diff
@@ -5980,7 +5980,7 @@ public abstract class Person : Participant {
    public string ssn { get; set; }
    public float height { get; set; }
    public System.DateTime dob { get; set; }
-   public Dictionary<string, KinTelephone> nextOfKin { get; set; }
+   public Dictionary<string, System.DateTime> nextOfKin { get; set; }
 }
```

Comment:
These types seem to have got more general. Is this expected?

Reply:
Yes, the underlying Primitive of a Scalar should be printed, in place of its alias name

Reply:
Beware the logic that handles UUID. Did you test with a map that has a UUID as a key or value? It looks like the code generator doesn't unbox those.

Reply:
@dselman logic added to handle `concerto.scalar.UUID` for k & v